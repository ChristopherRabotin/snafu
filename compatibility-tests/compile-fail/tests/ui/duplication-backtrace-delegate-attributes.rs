use snafu::prelude::*;

#[derive(Debug, Snafu)]
enum EnumError {
    AVariant {
        // Should detect second attribute as duplicate
        #[snafu(backtrace)]
        #[snafu(backtrace)]
        source: String,
    },
}

fn main() {}
