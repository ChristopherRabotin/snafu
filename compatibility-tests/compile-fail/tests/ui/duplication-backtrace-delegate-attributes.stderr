error: Multiple `backtrace` attributes are not supported on a field
 --> tests/ui/duplication-backtrace-delegate-attributes.rs:8:17
  |
8 |         #[snafu(backtrace)]
  |                 ^^^^^^^^^
